cypress/integration/mentors.js

```diff
@@ -1,11 +1,11 @@
 const { cy } = global;
 
 describe('Mentor Filtering', () => {
-  before(function() {
+  beforeAll(() => {
```

Comment:
Seems like it brakes the build - https://travis-ci.com/github/Coding-Coach/find-a-mentor/builds/211697891#L591

Reply:
Ah gotcha. Will see what's up with jest / cypress config. I would expect this to be defined.

Reply:
Put back the `before` function and now import it from global per other file so no undefined. Going to take a look at cypress setup, I think maybe we are missing jest or cypress config values.